src/main/java/io/jenkins/plugins/coverage/adapter/CoverageReportAdapter.java

```diff
@@ -0,0 +1,43 @@
+package io.jenkins.plugins.coverage.adapter;
+
+import hudson.ExtensionPoint;
+import hudson.model.Describable;
+import hudson.model.Descriptor;
+import io.jenkins.plugins.coverage.targets.CoverageResult;
+import jenkins.model.Jenkins;
+import org.w3c.dom.Document;
+
+import java.io.File;
+
+
+public abstract class CoverageReportAdapter implements ExtensionPoint, Describable<CoverageReportAdapter> {
```

Comment:
We should group declarations together - I think Jenkins projects in general use sun coding conventions http://www.oracle.com/technetwork/java/javase/documentation/codeconventions-141855.html#1852 - but @oleg-nenashev can verify

Reply:
What declarations should be grouped? I don't understand...Could you show me an example?

Reply:
I was mostly referring to variable and constructor order - here's what that doc says:

- instance variables - First public, then protected, then package level (no access modifier), and then private.
- Constructors
- These methods should be grouped by functionality rather than by scope or accessibility. For example, a private class method can be in between two public instance methods. The goal is to make reading and understanding the code easier.